cmd/controller/app/controllers.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package app
 
 import (
+	"kubesphere.io/devops/pkg/apiserver/authentication/token"
```

Comment:
You are still using golang packages from apiserver in the code base of a controller.

Reply:
Done. Please see https://github.com/kubesphere/ks-devops/pull/384/commits/2debd818f129aba4400ae4dd5d61591dcef57e37.